CODEOWNERS

```diff
@@ -0,0 +1 @@
+* sean.s.leblanc@gmail.com
```

Comment:
this bit will just make it so any pull requests that get opened are automatically assigned to me (feel free to add yourself too if wanted, i'm just adding it as a convenient way for me to get notified if other folks try to contribute)